src/templates/Templates.js

```diff
@@ -322,6 +322,11 @@ const AttributeTemplates = {
     attribute: "attribute",
     default: 1.0
   },
+  Supply: {
+    category: 'device',
```

Comment:
My only question is why every supply has a category of "device"?

Reply:
Sorry, I missed that you had reviewed this. Originally I was going to allow either "device" or "medication" as per the FHIR SupplyDelivery resource, but then I decided it wasn't worth doing that and changed it to just "device". I should just remove that altogether.